oxidation/libparsec/crates/platform_storage/src/sqlite/db/executor/mod.rs

```diff
@@ -115,7 +118,3 @@ impl SqliteExecutor {
         }).await.expect("blocking job task panicked")
     }
 }
-
-#[cfg(test)]
-#[path = "../tests/unit/executor.rs"]
-mod test;
```

Comment:
Why ? We said that we want to group tests in crate

Reply:
I understand that we prefer to not have the test in the same file, which is the case

Reply:
![image](https://user-images.githubusercontent.com/3187637/230039105-f1dbd149-6198-419b-a786-dce0e1dd3d11.png)

Reply:
done